js/controller.js

```diff
@@ -222,6 +227,25 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       }
     },
 
+    onHAWillFailover: function(){
+      this.state.failoverInProgress = true;
+      this.state.pauseAnimationDisabled = true;
+      this.renderSkin();
+    },
+
+    onHAFailoverComplete: function(){
+      this.state.failoverInProgress = false;
+      this.state.screenToShow = CONSTANTS.SCREEN.PLAYING_SCREEN;
+      this.renderSkin();
+    },
+
+    onHAFailoverError: function(){
+      this.state.failoverInProgress = false;
+      // this.state.screenToShow = CONSTANTS.SCREEN.ERROR_SCREEN;
```

Comment:
Which screen is shown when there is a failover error?

Reply:
none. It shows buffering for a few seconds.

Reply:
It will show error screen or whatever screen need to be show for displaying an error in network. HA plugin is not interfering with screens. HA will try to failover before the actual error got displayed. If it will failed to failover, then player should display screen that it should display in usual cases in case of network problems.